<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-JUA9SJ7" failures="2" tests="3" name="Test1" time="80.726" errors="0" timestamp="2025-09-11T16:06:10 IST">
  <testcase classname="testCases.TC003_LoginDDT" name="LoginWithDDT" time="20.048">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC003_LoginDDT.LoginWithDDT(TC003_LoginDDT.java:65)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- LoginWithDDT -->
  <testcase classname="testCases.TC003_LoginDDT" name="LoginWithDDT" time="8.708"/>
  <testcase classname="testCases.TC003_LoginDDT" name="LoginWithDDT" time="13.487">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC003_LoginDDT.LoginWithDDT(TC003_LoginDDT.java:65)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- LoginWithDDT -->
</testsuite> <!-- Test1 -->
